utils/getScenarioContent.tsx

```diff
@@ -1,75 +1,86 @@
 import { Claim, ClaimDetailsContent, ClaimStatusContent, ScenarioContent } from '../types/common'
 
-export type ScenarioTypeKey = keyof typeof ScenarioType
-
 export enum ScenarioType {
-  PendingDetermination = 'Pending determination scenario',
-  BasePending = 'Base state with pending weeks',
-  BaseNoPending = 'Base state with no pending weeks',
+  Scenario1,
+  Scenario7,
+  Scenario8,
+  Scenario9,
+  Scenario10,
+}
+
+export const ScenarioTypeNames = {
+  [ScenarioType.Scenario1]: 'Pending determination scenario',
+  [ScenarioType.Scenario7]: 'Base state; No pending weeks; No weeks to certify',
+  [ScenarioType.Scenario8]: 'Base state; No pending weeks; Has weeks to certify',
+  [ScenarioType.Scenario9]: 'Base state; Has pending weeks; No weeks to certify',
+  [ScenarioType.Scenario10]: 'Base state; Has pending weeks; Has weeks to certify',
```

Comment:
Note: This is the major change. After speaking with Nicole and Diana today, we agreed to call scenarios by number instead of a nick name. As part of that, I converted the enum to a numeric enum, which is able to do automatic [reverse mappings](https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings). Enums are a little finicky with Typescript, so I added the `numericEnum.tsx` utility to help.

Reply:
I like it. Can we add a comment to the code that points to the fully-written-out Scenario descriptions?

Reply:
@lomky Yes for sure. But I'm not sure exactly where you are thinking we should add the comment? In `getScenarioContent.tsx`?

Reply:
I was thinking right above this const, or if we do topline comments on the files, there would also be good!